src/status_im/multiaccounts/login/core.cljs

```diff
@@ -252,6 +252,17 @@
   [cofx]
   {::initialize-communities-enabled nil})
 
+(fx/defn get-node-config-callback
+  {:events [::get-node-config-callback]}
+  [{:keys [db] :as cofx} node-config]
+  (fx/merge cofx
```

Comment:
You don't need to `fx/merge` here,
just
```
{:db ....}
```
will work

Reply:
Done.